cbf/v1/cbf-v1.0.0.yaml

```diff
@@ -0,0 +1,556 @@
+              examples:
+                Example 1:
+                  value:
+                    httpMethod: GET
+                    requestUri: 'https://dcsa.org/dcsa/dcsa-standard/v1/booking-forecast'
+                    statusCode: 400
+                    statusCodeText: Bad Request
+                    errorMessage: The supplied data could not be accepted
+                    providerCorrelationID: 4426d965-0dd8-4005-8c63-dc68b01c4962
+                    errorDateTime: '2019-11-12T07:41:00+08:30'
+                    errors:
+                      - errorCode: 7003
+                        field: location.facilityCode
+                        value: SG SIN WHS
+                        reason: invalidQuery
+                        message: Spaces not allowed in facility code
+        '500':
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                $ref: ../../models/ErrorResponse.yaml
+      operationId: create-v1-booking-forecast
+      description: Submits a Booking forecast.
+      parameters:
+        - $ref: ../../models/Headers.yaml#/components/parameters/APIVersionHeader
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/BookingForecast'
+        description: ''
+components:
+  schemas:
+    BookingForecast:
+      title: BookingForecast
+      x-stoplight:
+        id: keuoy8j4qkve1
+      type: object
+      properties:
+        partnerIdentity:
+          $ref: '#/components/schemas/PartnerIdentity'
+        carrierServiceName:
+          type: string
+          pattern: ^\S+(\s+\S+)*$
+          maxLength: 50
+          example: Great Lion Service
+          description: The name of a service as specified by the carrier
+        carrierServiceCode:
+          type: string
+          pattern: ^\S+(\s+\S+)*$
+          maxLength: 11
+          example: FE1
+          description: The carrier-specific code of the service for which the schedule details are published.
+        universalServiceReference:
+          type: string
+          description: 'A global unique service reference, as per DCSA standard, agreed by VSA partners for the service. The service reference must match the regular expression pattern: SR\d{5}[A-Z]. The letters SR followed by 5 digits, followed by a checksum-character as a capital letter from A to Z.'
+          example: SR12345A
+          maxLength: 8
+          pattern: '^SR\d{5}[A-Z]$'
+        vessels:
+          type: array
+          items:
+            $ref: '#/components/schemas/Vessel'
+      required:
+        - partnerIdentity
+        - carrierServiceName
+        - carrierServiceCode
+        - vessels
+    PartnerIdentity:
+      title: PartnerIdentity
+      x-stoplight:
+        id: mnr99wlnor6is
+      type: object
+      properties:
+        code:
+          type: string
+          format: regex
```

Comment:
I don't think a regex format exists - this would "just" be string - the pattern would define the regex